ReelTalk/containers/Lolomo.js

```diff
@@ -24,12 +24,9 @@ var Lolomo = React.createClass({
 
   _showDetails: function(show) {
     this.props.navigator.push({
-      title: show.name,
+      title: 'Show',
```

Comment:
@BillyBarbaro it may be ambiguous but no more ambiguous/confusing than labeling a movie as a show. Plus, in the design mocks this page doesn't even have a header so it would be just for internal naming. And also then it would allow us to have more specific names for attributes like movies and shows which may contain different information. For example shows may list info about seasons and episodes while movies will not.